.gitignore

```diff
@@ -1,2 +1,30 @@
-node_modules
-.DS_Store
\ No newline at end of file
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+
+# testing
+/coverage
+cypress/videos/
+
+# production
+/build
+.npmrc
+dist
```

Comment:
I think we should remove this from .gitignore since we'll want people to include the production/dist files in the repo, similar to how we've done other add-ons.